sleepecg/heartbeats.py

```diff
@@ -626,11 +625,10 @@ def _thresholding_py(
                     threshold_F1 /= 2
                     threshold_I1 /= 2
 
-            # A signal peak has been found, so performing a searchback
-            # makes sense.
+            # a signal peak has been found, so perform a searchback
```

Comment:
`do_searchback` being `True` is just a necessary requirement for performing a searchback, not a sufficient one (see line 452). 
```suggestion
            # A signal peak has been found, so performing a searchback
            # makes sense.
```